app/controllers/application_controller.rb

```diff
@@ -36,8 +36,6 @@ def send_email(params)
     response = sg.client.mail._('send').post(request_body: mail.to_json)
     if response.status_code == "202"
       [202, ({message: "Message sent successfully"}).to_json]
-    else
-      [400, ({message: "There was an error proccessing your request"}).to_json]
     end
```

Comment:
I think if we leave the if statement, and sengrid sends us anything other than a 202, the microservice would break. Do we want to just remove the if statement and send back line 38 with the response.status_code as the status instead of a hard coded 202? Then BE can handle error recovery.

Reply:
Fixed!